DuckDuckGo/PrivacyProtectionOverviewController.swift

```diff
@@ -100,22 +94,13 @@ class PrivacyProtectionOverviewController: UITableViewController {
         trackersCell.summaryLabel.text = siteRating.networksText(contentBlocker: contentBlocker)
 
         if protecting() || siteRating.uniqueTrackersDetected == 0 {
-            trackersCell.summaryImage.image = #imageLiteral(resourceName: "PP Icon Networks On")
```

Comment:
These icons are still in the project. I believe they are now unused - if so, let's delete them